### ISSUE TITLE
18F website improvements

### ISSUE THREAD

**Issue by MelissaBraxton** [18F/TLC-crew#313]
### A description of the work
This issue covers designing and implementing refinements to the 18F website. 

### Point of contact on this issue
@quarterback 

### Reproduction steps (if necessary)
Be as specific as possible

**Billable?**

- [ ] Yes
- [X] No
If yes, tock code:

**Skills needed**

- [ ] Any human
- [X] Design
- [ ] Content
- [X] Engineering
- [ ] Acquisition
- [ ] Product
- [ ] Other

**Timeline**
Does this need to happen in the next two weeks?

- [ ] Yes
- [X] No

How much time do you anticipate this work taking?

### Acceptance Criteria

- [ ] We have implemented refinements to the 18F website. 

### Possible tasks
- [X] Mock up refinements
- [ ]  Set up staging site 
- [ ]  Submit a PR to implement new design
- [ ] Conduct user research to inform further site refinements (likely needs to be its own issue)

**Comment by durland18F** [comment]
Hiya!

I think there are really two phases to this work, because my understanding is that there is a mandate to get the content changes up quickly:
1. Implementing the content changes to a staging server, making any necessary front-end code adjustments to accommodate the lighter content, moving the changes to production
2. Implementing some version of the design changes to staging, pushing to production

If instead we wanted to make all of the changes it might be more efficient but slower to show progress.

**Comment by MelissaBraxton** [comment]
Sounds good! @durland18F - Do you want to take a crack at splitting this out into two issues?

**Comment by MelissaBraxton** [comment]
I believe @durland18F is writing up an issue that will precede this one / or more issues that will comprise this work.

**Comment by MelissaBraxton** [comment]
Moving this to where we keep our epics with the understanding that @austinhernandez will be writing up a new issue in the increment goal for the first step here that could be achieved in the 7/5 increment.

**Comment by austinhernandez** [comment]
Rolling off TLC (for now) and going thru all my open tickets. I can give help on this ticket if needed, but I think this is mostly for Mel and Igor. Most of the other 18F website tickets are in the icebox, but this one was still open so I wanted to give a heads up to @MelissaBraxton and @alexbielen.

**Comment by alexbielen** [comment]
@austinhernandez yup, that sounds good. I think that @igorkorenfeld is working on it from here so I'll remove you and @melchoyce . Thanks for the heads up!
